src/components/blog/BlogOverview.jsx

```diff
@@ -1,5 +1,42 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import { Container, Row } from "react-bootstrap";
+import SingleBlogCard from "./SingleBlogCard";
+import "./style.overview.scss";
+import BlogHeader from "./BlogHeader";
 
 export default function BlogOverview() {
-  return <div></div>;
+  const [allBlogs, setAllBlogs] = useState([]);
```

Comment:
you probably want a different noun here, blog refers to a blog as a whole so this makes it sound like you have 3-4 blogs. posts i would guess